src/merc.h

```diff
@@ -1992,249 +1989,248 @@ extern int max_on;
-int fread_spnumber args((FILE * fp));
-long fread_flag args((FILE * fp));
+void boot_db(void);
+void area_update(void);
+CD *create_mobile(MOB_INDEX_DATA *pMobIndex);
+void clone_mobile(CHAR_DATA *parent, CHAR_DATA *clone);
+OD *create_object(OBJ_INDEX_DATA *pObjIndex, int level);
+void clone_object(OBJ_DATA *parent, OBJ_DATA *clone);
+void clear_char(CHAR_DATA *ch);
+void free_char(CHAR_DATA *ch);
+char *get_extra_descr(const char *name, EXTRA_DESCR_DATA *ed);
+MID *get_mob_index(int vnum);
+OID *get_obj_index(int vnum);
+RID *get_room_index(int vnum);
+char fread_letter(FILE *fp);
+int fread_number(FILE *fp);
+int fread_spnumber(FILE *fp);
+long fread_flag(FILE *fp);
 BUFFER *fread_string_tobuffer(FILE *fp);
-char *fread_string args((FILE * fp));
-char *fread_string_eol args((FILE * fp));
-void fread_to_eol args((FILE * fp));
-char *fread_word args((FILE * fp));
-long flag_convert args((char letter));
-void *alloc_mem args((int sMem));
-void *alloc_perm args((int sMem));
-void free_mem args((void *pMem, int sMem));
-char *str_dup args((const char *str));
-void free_string args((char *pstr));
-int number_fuzzy args((int number));
-int number_range args((int from, int to));
-int number_percent args((void));
-int number_door args((void));
-int number_bits args((int width));
-int number_mm args((void));
-int dice args((int number, int size));
-int interpolate args((int level, int value_00, int value_32));
-void smash_tilde args((char *str));
-bool str_cmp args((const char *astr, const char *bstr));
-bool str_prefix args((const char *astr, const char *bstr));
-bool str_infix args((const char *astr, const char *bstr));
-bool str_suffix args((const char *astr, const char *bstr));
-char *capitalize args((const char *str));
-void append_file args((CHAR_DATA * ch, char *file, char *str));
-void bug args((const char *str, ...)) __attribute__((format(printf, 1, 2)));
-void log_string args((const char *str));
-void log_new args((const char *str, int loglevel, int level));
-void tail_chain args((void));
+char *fread_string(FILE *fp);
+char *fread_string_eol(FILE *fp);
+void fread_to_eol(FILE *fp);
+char *fread_word(FILE *fp);
+long flag_convert(char letter);
+void *alloc_mem(int sMem);
+void *alloc_perm(int sMem);
+void free_mem(void *pMem, int sMem);
+char *str_dup(const char *str);
+void free_string(char *pstr);
+int number_fuzzy(int number);
+int number_range(int from, int to);
+int number_percent(void);
+int number_door(void);
+int number_bits(int width);
+int number_mm(void);
+int dice(int number, int size);
+int interpolate(int level, int value_00, int value_32);
+void smash_tilde(char *str);
+bool str_cmp(const char *astr, const char *bstr);
+bool str_prefix(const char *astr, const char *bstr);
+bool str_infix(const char *astr, const char *bstr);
+bool str_suffix(const char *astr, const char *bstr);
+char *capitalize(const char *str);
+void append_file(CHAR_DATA *ch, char *file, char *str);
+void bug(const char *str, ...);
```

Comment:
Can you please restore the `__attribute__((format(printf, 1, 2)));` bit here? Although I've fixed all the `bug()`s it would be a shame to lose this.